oneview_redfish_toolkit/conf/redfish.conf

```diff
@@ -38,6 +38,7 @@ organizationName = Hewlett Packard Enterprise
 ServiceRoot = ServiceRoot.v1_2_0.json
 ChassisCollection = ChassisCollection.json
 Chassis = Chassis.v1_5_0.json
+CollectionCapabilities = CollectionCapabilities.v1_0_0.json
```

Comment:
Do we still need to include this entry for CollectionCapabilities?

Reply:
Done.